packages/datetime/src/dateRangeSelectionStrategy.ts

```diff
@@ -0,0 +1,155 @@
+
+import {
+    areSameDay,
+    DateRange,
+    DateRangeBoundary,
+} from "./common/dateUtils";
+
+export interface IDateRangeSelectionState {
```

Comment:
oh it's the return type of the functions! can you put it up front in the signature instead of casting at the end?

`public static getNextState(args): IDateRangeSelectionState { }`